include/z64player.h

```diff
@@ -877,6 +877,7 @@ typedef struct Player {
 
     /* 0x084F */ union {
         s8 actionVar1;
+        s8 slideFacingUpSlope; // Player_Action_SlideOnSlope: whether link is sliding facing up the slope or not
```

Comment:
if you take my suggestion of `facingUpSlope`, I think the comment can be 
`// Player_Action_SlideOnSlope: facing uphill when sliding on a slope`

Reply:
Done.